app/controllers/regulatory-attachments/edit.js

```diff
@@ -279,4 +282,39 @@ export default class EditController extends Controller {
     await documentContainer.save();
     this._editorDocument = editorDocument;
   });
+
+  get documentSnippetListIds() {
+    return (
+      this.editor
+        .getDocumentAttribute('snippet-list-ids')
+        ?.split(',')
+        .filter(Boolean) ?? []
+    );
+  }
+
+  set documentSnippetListIds(snippetIds) {
+    this.editor.setDocumentAttribute('snippet-list-ids', snippetIds.join(','));
+    this.assignedSnippetListsIds = snippetIds;
+  }
+
+  setDocumentContainerSnippetLists = task(async (snippetIds) => {
+    if (!snippetIds || !snippetIds.length) {
+      this.documentSnippetListIds = [];
+      this.model.documentContainer.snippetLists.setObjects([]);
+
+      return this.save.perform();
+    }
+
+    const snippetLists = await this.store.query('snippet-list', {
```

Comment:
I don't understand why this works... you dont provide any pageSize param but it does pull all the lists afaict

I can't see a change to the default pagesize in the resources config in the related PR either :thinking: 

Anyway would be nice to at least sort these in some way. 
Later we should also add proper pagination here, and sortable datatable columns, but lets not block this pr for that